.travis.install.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
 set -x
```

Comment:
http://unix.stackexchange.com/questions/121415/shell-script-header-for-best-compatibility are you sure, even after reading that?